blob/blob.go

```diff
@@ -867,7 +867,7 @@ type WriterOptions struct {
 	BeforeWrite func(asFunc func(interface{}) bool) error
 }
 
-// A type that implements BucketURLOpener can open buckets based on a URL.
+// BucketURLOpener represents types that can open buckets based on a URL.
```

Comment:
I'm curious why this changed. If the move is toward making the type be the first word of the sentence, it's worth interface type doc comments sometimes follow this format this exists. See [`sort.Interface`](https://golang.org/pkg/sort/#Interface)

Reply:
I changed it because `golint` complains:

```
$ golint
blob.go:870:1: comment on exported type BucketURLOpener should be of the form "BucketURLOpener ..." (with optional leading article)
```